hack/jenkins/job-configs/kubernetes-test-history.yaml

```diff
@@ -0,0 +1,15 @@
+- job:
+    name: kubernetes-test-summary
+    description: 'Create a daily test summary and upload to GCS. Test owner: spxtr.'
+    triggers:
+        - timed: 'H 0 * * *'
```

Comment:
Since H isn't in standard cron format can you link to documentation of this feature in jenkins in a comment?